src/common/transformations/tests/op_conversions/convert_nms_to_nms_ie_internal_test.cpp

```diff
@@ -26,52 +26,6 @@
 using namespace testing;
 using namespace ov;
 
-TEST_F(TransformationTestsF, ConvertNMS1ToNMSIEInternal) {
-    {
-        auto boxes = std::make_shared<opset1::Parameter>(element::f32, Shape{1, 1000, 4});
-        auto scores = std::make_shared<opset1::Parameter>(element::f32, Shape{1, 1, 1000});
-        auto max_output_boxes_per_class = opset1::Constant::create(element::i64, Shape{}, {10});
-        auto iou_threshold = opset1::Constant::create(element::f32, Shape{}, {0.75});
-        auto score_threshold = opset1::Constant::create(element::f32, Shape{}, {0.7});
-        auto nms = std::make_shared<opset1::NonMaxSuppression>(boxes,
-                                                               scores,
-                                                               max_output_boxes_per_class,
-                                                               iou_threshold,
-                                                               score_threshold,
-                                                               op::v1::NonMaxSuppression::BoxEncodingType::CORNER,
-                                                               true);
-
-        model = std::make_shared<Model>(NodeVector{nms}, ParameterVector{boxes, scores});
-
-        manager.register_pass<ov::pass::ConvertNMS1ToNMS5>();
-        manager.register_pass<ov::pass::ConvertNMSToNMSIEInternal>();
-        manager.register_pass<pass::ConstantFolding>();
-
-        // as inside test infrastructure we can not predict output names for given Model
-        // we have to enable soft names comparison manually
-        enable_soft_names_comparison();
```

Comment:
seems like we've lost a little bit of coverage. can we still have a test without name comparison?

Reply:
we still have in convert**_nms9**_to_nms_ie_internal_test.cpp
But we are going to delete this test in the next Release, so it's better to create a new test without nms specifics in the next PR